plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -110,6 +110,10 @@ export default class ChatChannel {
     ).length;
   }
 
+  get unreadThreadsCount() {
+    return Array.from(this.threadsManager.unreadThreadOverview.values()).length;
```

Comment:
This is a `Map` (well `TrackedMap`), so you should be able to use `.size` and not have to convert to array https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/size